src/routes/safe/components/Apps/components/AddAppForm/index.tsx

```diff
@@ -19,6 +18,22 @@ const FORM_ID = 'add-apps-form'
 const StyledTextFileAppName = styled(TextField)`
   && {
     width: 385px;
+    .MuiFormLabel-root {
+      &.Mui-disabled {
+        color: rgba(0, 0, 0, 0.54);
+        &.Mui-error {
+          color: ${theme.colors.error};
```

Comment:
Isn't the theme passed via props like here https://styled-components.com/docs/advanced#theming?

Reply:
Sure!! Updated it